src/tests/system/tests/test_ldap.py

```diff
@@ -34,7 +34,7 @@ def test_ldap__change_password(client: Client, ldap: LDAP, modify_mode: str):
         2. Password is changed successfully
         3. User is authenticated
         4. User is not authenticated
-    :customerscenario: False
+    :customerscenario: True
```

Comment:
If this change intentional? If so, can you also add the ticket?

Reply:
Yes, this change was intentional. I forgot to insert the bg ticket. FIXED